provisioners/deploy-sftp.yml

```diff
@@ -41,13 +41,15 @@
         state: stopped
       loop:
         - sftp.service
+        - sftp-storage-cleanup.timer
     - name: Start all services
       service:
         name: "{{ item }}"
         enabled: yes
         state: started
       loop:
         - sftp.service
+        - sftp-storage-cleanup.timer
```

Comment:
I am not entirely sure the difference between deploy and configure -- I imagine configure is run once and deploy is run every time there is a new deployment?  If that is the case, I think I should probably remove the changes to this file, since the timer / service does not ever need to be restarted.

Reply:
I think that's right.